### PR TITLE
Combine the agile content under one folder

### PR DESCRIPTION
**Pull request by TessFerrandez**
Move the following sections under the agile development folder

- backlog management
- retrospectives
- scrum of scrums
- sprint planning
- standups
- team agreements

Create a basic README to point to the sections

Add a link to said README from the main README

Update any links that point to agile content on other pages

### CODE REVIEW

**Comment by meulta** [comment]
LGTM

File: agile-development/stand-ups/readme.md [hunk 1]
```diff
@@ -1,110 +1,110 @@
...
+
+Make a best effort to begin answering the 3 questions as close to the scheduled start time as possible. Try not to waste time upfront on chit-chat or waiting on all team members to join. This can extend the meeting time significantly. Starting immediately will help ensure stand-ups remain effective and useful over time.
+
+### Same Time Everyday
+
+Stand-up should be held at the same time each day. The meeting time should be mutually agreed-upon by the contributors, and should take into consideration time zones, working schedules, and other factors so that every team member can reasonably participate.
+
+### Schedule Stand-ups for Mid-Morning
+
+```plaintext
```
Comment (bsherwin) [inline agile-development/stand-ups/readme.md:93]: Can we get rid of this coffee cup?

File: agile-development/stand-ups/readme.md [hunk 2]
```diff
@@ -1,110 +1,110 @@
...
+            ((((
+             ))))
+          _ .---.
+         ( |`---'|
+          \|     |
+          : .___, :
+           `-----'
+```
+
+Many teams find that the ideal time for stand-ups is between 9 and 10am. This provides ample time for team members to get into the office, settle in (`grab a coffee ;)`), and catch up with where they left off the day before. It's also early enough to help developers to plan their day around new work items.
```
Comment (bsherwin) [inline agile-development/stand-ups/readme.md:104]: Can we remove "(`grab a coffee;)`?

File: agile-development/stand-ups/readme.md (hunk 1 above)
Comment (TessFerrandez) [reply agile-development/stand-ups/readme.md:93]: For context, I just moved the existing text, but happy to remove the coffee cup :)

File: agile-development/stand-ups/readme.md (hunk 2 above)
Comment (TessFerrandez) [reply agile-development/stand-ups/readme.md:104]: Same as above - will remove

**Pull request merged**